pkg/daemon/ceph/client/osd.go

```diff
@@ -180,6 +180,26 @@ func EnableScrubbing(context *clusterd.Context, clusterName string) (string, err
 	return string(buf), nil
 }
 
+//SetNoOut execs `ceph osd set noout`
```

Comment:
```suggestion
// SetNoOut execs `ceph osd set noout`
```

Reply:
Updating all comments to start with a space.